sql/src/main/java/org/apache/druid/sql/calcite/external/ExternalOperatorConversion.java

```diff
@@ -120,7 +121,9 @@ public ExternalTableSpec apply(
         );
       }
       catch (JsonProcessingException e) {
-        throw new RuntimeException(e);
+        throw DruidException.forPersona(DruidException.Persona.USER)
+                            .ofCategory(DruidException.Category.INVALID_INPUT)
+                            .build(e, e.getMessage());
```

Comment:
For this one specifically, just so you know, there is an `InvalidInput.exception()` that you can use.  This makes it part of the `"invalidInput"` group of exceptions instead of the `"general"` group of exceptions.